be/src/vec/exec/file_hdfs_scanner.cpp

```diff
@@ -28,6 +28,10 @@ ParquetFileHdfsScanner::ParquetFileHdfsScanner(RuntimeState* state, RuntimeProfi
                                                ScannerCounter* counter)
         : HdfsFileScanner(state, profile, params, ranges, pre_filter_texprs, counter) {}
 
+ParquetFileHdfsScanner::~ParquetFileHdfsScanner() {
+    ParquetFileHdfsScanner::close();
```

Comment:
I think we should move this call to `FileScanNode` when this `FileScanner` is ready to close.

Reply:
I think FileScanNode in this version we can keep the same with before